### ISSUE TITLE
Use different default port

### ISSUE THREAD

**Issue by mpdaugherty** [weaviate/typescript-embedded#5]
Port 6666 is considered a "bad port" by a number of libraries, including the undici implementation in Node.js.

* [Fetch standard bad ports](https://fetch.spec.whatwg.org/#block-bad-port)
* [Undici bad port list](https://github.com/nodejs/undici/blob/5c3c69400ef46fac25a798fdf3dc6d422ef8b79e/lib/fetch/constants.js#L17)
* [Chrome blocked ports](https://www-archive.mozilla.org/projects/netlib/portbanning#portlist)

When trying to use the `.get()` api with Weaviate embedded in Node on MacOS, I get a `Fetch Failed: Bad Port` error - which means the fetch request is blocked before it is even attempted. I've semi-worked around this by using Axios and constructing GraphQL queries manually, but I'd prefer to use the Weaviate JS library.

I've also tried specifying the port number when setting up Weaviate Embedded; for some reason that has not helped. I haven't debugged all the way through that - if there ends up being a bug there I can create a separate issue. This issue is just focused on the suggestion to use a different default port than one that is blocked in multiple libraries.

**Comment by mpdaugherty** [comment]
FYI, in case helpful. Here's my config:

```javascript
weaviatePort = 7001;
  console.log(`Using port ${weaviatePort} for Weaviate`);
  weaviateHost = `127.0.0.1:${weaviatePort}`;
  console.log(weaviateHost)

  client = new WeaviateClient(
    new EmbeddedOptions({
      env: {
        ENABLE_MODULES: "text2vec-openai",
        DEFAULT_VECTORIZER_MODULE: "text2vec-openai",
        OPENAI_APIKEY: process.env.OPENAI_APIKEY,
      },
      persistence_data_path: DATA_PATH,
      port: weaviatePort
    },
    {
      scheme: 'http',
      host: weaviateHost
    })
  );
  ```
  
  And some code that fails:
  
  
  ```javascript
  const client = await weaviate.getClient();
      const response = await client.graphql.get()
        .withClassName('Message').withFields('messageId')
        .do();
```

**Comment by KrishnaManohar1997** [comment]
Facing this same issue, I don't think there's any update on this issue.
@mpdaugherty could you share the workaround queries to make this work, Thanks~!

**Comment by mpdaugherty** [comment]
@KrishnaManohar1997 I ended up monkey patching fetch:

```
// This monkey patch is for weaviate. Embedded weaviate's client always uses port 6666, which
// is blocked by Node's default fetch implementation, undici.
const nodeFetch = require('node-fetch');
global.fetch = nodeFetch;
```
